assets/pages/bundles-landing/bundlesLanding.scss

```diff
@@ -1,30 +1,131 @@
 #bundles-landing-page {
 
-	// ----- Page Components ----- //
+  // ----- Page Components ----- //
 
-	.bundles__bundle {
-		width: 300px;
-		padding: 8px 10px 12px;
-		float: left;
-	}
+  .bundles__bundle {
+    width: 300px;
+    padding: 8px 10px 12px;
+    float: left;
+  }
 
-	.bundles__bundle--digital {
-		background-color: #1db7d7;
-	}
+  .bundles__bundle--digital {
+      background-color: #1db7d7;
+  }
 
-	.bundles__bundle--paper {
-		background-color: #58a1ce;
-	}
+  .bundles__bundle--paper {
+      background-color: #58a1ce;
+  }
 
-	// ----- Shared Components ----- //
+  .ways-of-support {
+    max-width: 980px;
```

Comment:
It sounds fair, I will add that function to `breakpoint.css` and from now on, we will be able to use it.